### DISCUSSION TITLE
Velero datamover and node restrictions

### DISCUSSION THREAD

**Discussion by groucho64738** [vmware-tanzu/velero#7185] [Community support Q&A]
We have a setup (not sure how unique it is) where we've got some storageclasses that are only available on certain physical nodes in our kubernetes stack. I was trying to get velero v1.12 to do its datamover operation to a minio S3 bucket that we have set up for this. We have the node agent installed and it installs on ALL nodes in the cluster (which is fine as there could be uses other than just mine), but it seems like the datamover process kicked off a container which attempts to mount (I'm guessing) the snapshot, and that container ended up on a node that would not be able to mount that snapshot. 

I guess my question from that convoluted explanation is, is it possible to limit which nodes the datamover pod starts up on?

I might investigate trying to install velero just to use those few nodes and confirm that this is the case, rather than the default install across the entire cluster. 

Thanks.

**Comment by Lyndon-Li** [comment]
Not now, but may be in a future release, see #7036
